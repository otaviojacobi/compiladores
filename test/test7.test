
int main() {
    char a = '\n'; // segundo https://github.com/schnorr/comp/issues/37 podemos definir como implementar isso
                   // fiz como em C :)
    string b = "\"o\ti\"";
    b += a;

    return 0;
}