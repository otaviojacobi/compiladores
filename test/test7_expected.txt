2 TK_PR_INT [int]
2 TK_IDENTIFICADOR [main]
2 TK_ESPECIAL [(]
2 TK_ESPECIAL [)]
2 TK_ESPECIAL [{]
3 TK_PR_CHAR [char]
3 TK_IDENTIFICADOR [a]
3 TK_ESPECIAL [=]
3 TK_LIT_CHAR ['\n']
3 TK_ESPECIAL [;]
5 TK_PR_STRING [string]
5 TK_IDENTIFICADOR [b]
5 TK_ESPECIAL [=]
5 TK_LIT_STRING ["\"o\ti\""]
5 TK_ESPECIAL [;]
6 TK_IDENTIFICADOR [b]
6 TK_ESPECIAL [+]
6 TK_ESPECIAL [=]
6 TK_IDENTIFICADOR [a]
6 TK_ESPECIAL [;]
8 TK_PR_RETURN [return]
8 TK_LIT_INT [0]
8 TK_ESPECIAL [;]
9 TK_ESPECIAL [}]